Sources/Spectre/XCTest.swift

```diff
@@ -32,7 +32,13 @@ class XcodeReporter: ContextReporter {
   func addDisabled(_ name: String) {}
 
   func addFailure(_ name: String, failure: FailureType) {
-    testCase.recordFailure(withDescription: "\(name): \(failure.reason)", inFile: failure.file, atLine: failure.line, expected: false)
+    if #available(OSX 10.13, *) {
```

Comment:
I don't quite understand you would want to write this check for a certain macOS being available. From the Xcode documentation it states that `recordFailure` is deprecated in Xcode 12, and that `record()` is available from Xcode 12 onwards. Should this check instead be against Xcode 12 and not a particular operating system (High Sierra)?

This change appears to break building with Xcode 10/11, (tested against macOS 10.14 and 10.15) the following build errors are emitted:

```
/Users/runner/work/Spectre/Spectre/Sources/Spectre/XCTest.swift:38:28: error: use of unresolved identifier 'XCTSourceCodeLocation'
            let location = XCTSourceCodeLocation(filePath: failure.file, lineNumber: failure.line)
                           ^~~~~~~~~~~~~~~~~~~~~
/Users/runner/work/Spectre/Spectre/Sources/Spectre/XCTest.swift:39:25: error: use of unresolved identifier 'XCTIssue'
            let issue = XCTIssue(type: .assertionFailure, compactDescription: "\(name): \(failure.reason)", detailedDescription: nil, sourceCodeContext: .init(location: location), associatedError: nil, attachments: [])
                        ^~~~~~~~

Testing failed:

	Use of unresolved identifier 'XCTSourceCodeLocation'

```

Reply:
here I replaced the `testCase.recordFailure` by `testCase.record` as the documentation suggests, maybe you need to see what is the best os condition to work on every situation,

I'm not happy to have a answer after 24 days, so if you can't maintain this repository, the best solution is to close it and tell all the projects use this dependancy to not use it,